docs/content/modeling/testing-models.mdx

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 6
+slug: /modeling/testing
+description: Testing Models
+---
+
+import {
+  DocumentationNotice,
+  ProductName,
+  ProductNameFormat,
+  RelatedSection,
+} from '@components/Docs';
+
+# Testing Models
+
+<DocumentationNotice />
+
+You must test every <ProductName format={ProductNameFormat.ShortForm}/> model before deployment to ensure your authorization model is correctly designed.
+
+The `.fga.yaml` file provides a way to test <ProductName format={ProductNameFormat.ShortForm}/> authorization models. If you are using Visual Studio Code as your IDE, install the [OpenFGA extension](https://marketplace.visualstudio.com/items?itemName=openfga.openfga-vscode), to enable syntax coloring and validation.
```

Comment:
`provides a way to test` can just be `tests`.

"If you use" flows a little better than "if you are using". 

The comma before `to enable syntax` is probably unnecessary.